src/java/org/apache/cassandra/config/DatabaseDescriptor.java

```diff
@@ -5349,4 +5349,28 @@ public static boolean isPasswordValidatorReconfigurationEnabled()
     {
         return conf.password_validator_reconfiguration_enabled;
     }
+
+    public static long getInvalidClientDisconnectPeriod(TimeUnit unit)
+    {
+        return conf.invalid_client_disconnect_period.to(unit);
+    }
+
+    public static void setInvalidClientDisconnectPeriod(TimeUnit unit, long duration)
+    {
+        long prevDuration = conf.invalid_client_disconnect_period.to(unit);
+        logger.info("invalid_client_disconnect_period set from {} to {}", prevDuration, duration == 0 ? "disabled" : String.format("%s %s", duration, unit));
```

Comment:
NIT to avoid the string concat, although we don't save much here
```suggestion
        logger.info("invalid_client_disconnect_period set from {} to {} ({})", prevDuration, duration == 0 ? "disabled" : duration, unit);
```

The unit shows all uppercase but that's fine, I don't want to do a lowercase here.

Reply:
This would render as "disabled (MILLISECONDS)" though. This is going to log super infrequently, so clarity feels more important than performance.